packages/desktop-client/src/components/mobile/accounts/AccountTransactions.tsx

```diff
@@ -40,14 +46,24 @@ import { MobileBackButton } from '../MobileBackButton';
 import { AddTransactionButton } from '../transactions/AddTransactionButton';
 import { TransactionListWithBalances } from '../transactions/TransactionListWithBalances';
 
-export function AccountTransactions({ account, pending, failed }) {
+import { type AccountViewModel } from './AccountViewModel';
+
+export function AccountTransactions({
+  account,
+}: {
+  readonly account: AccountViewModel;
```

Comment:
We haven't been using much classes / view models in the project at the moment. Can we instead just create a type for the these props?